cypress/e2e/spaces/spaces.spec.ts

```diff
@@ -274,8 +302,12 @@ describe("Spaces", () => {
         cy.checkA11y(undefined, axeOptions);
         cy.get(".mx_SpacePanel").percySnapshotElement("Space panel collapsed", { widths: [68] });
 
-        cy.get(".mx_SpaceButton_toggleCollapse").click({ force: true });
-        cy.get(".mx_SpacePanel:not(.collapsed)").should("exist");
+        cy.findByRole("tree", { name: "Spaces" }).within(() => {
+            // This finds the expand button with the class name "mx_SpaceButton_toggleCollapse". Note there is another
+            // button with the same name with different class name "mx_SpacePanel_toggleCollapse".
```

Comment:
This comment is a bit confusing, is the second class meant to be different?

Reply:
It is intended to note that there are two buttons whose name is `Expand` and it is not possible to select the proper one without specifying with `findByRole("tree", { name: "Spaces" }).within`.